custom_components/solarman/inverter_definitions/deye_sg01hp3.yaml

```diff
@@ -315,7 +315,7 @@ parameters:
         configurable:
           min: 0
           max: 240
-        icon: "mdi:battery"
+        icon: "mdi:current-dc"
```

Comment:
@davidrapan changed all "Charging Current" entities to have a better representing icon